bin/asciibinder

```diff
@@ -155,10 +182,23 @@ if (not repo_dir.nil? and not ARGV.empty?) or (repo_dir.nil? and ARGV.length > 1
   Trollop::die "Too many arguments provided to ascii_binder: '#{ARGV.join(' ')}'. Exiting."
 elsif repo_dir.nil?
   if ARGV.length == 1
-    repo_dir = Pathname.new(ARGV.shift)
+    if cmd == 'clone'
+      cmd_opts[:giturl] = ARGV.shift
+      if cmd_opts[:dir] != ''
+        repo_dir = Pathname.new(cmd_opts[:dir])
+      else
+        repo_dir = File.join(Pathname.pwd, cmd_opts[:giturl].split('/')[-1].split('.')[0])
```

Comment:
FYI, Pathname object have the join method directly, so you could do something like...

```ruby
Pathname.pwd.join(cmd_opts[:giturl].split('/')[-1].split('.')[0])
```

Not going to hold up the PR for that though.